apps/backoffice-v2/src/components/pages/Individuals/Individuals.route.tsx

```diff
@@ -33,10 +35,8 @@ const BusinessesSearchSchema = SearchSchema.extend({
   sortBy: z.enum(['website', 'address']).optional().catch('website'),
   filter: z
     .object({
-      approvalState: z
-        .array(z.enum([States]))
-        .optional()
-        .catch([]),
+      approvalState: z.array(z.enum(States)).optional().catch([]),
```

Comment:
Fixed an unrelated bug here: changed from `z.enum([States])` to `z.enum(States)` (not wrapping `States` with an array)